vonk/configuration/logsettings.rst

```diff
@@ -72,98 +73,113 @@ But in this (purposefully incorrect) example the ``Warning`` level on the ``Vonk
 Changing the sink
 -----------------
 Another setting you can adjust is ``WriteTo``. This tells Serilog which sink(s) to log to.
-Serilog provides several sinks, and for Vonk you can use ``Console``, ``ColoredConsole``, ``RollingFile`` and ``Seq``.
+Serilog provides several sinks, and for Vonk you can use ``Console``, ``File``, ``ApplicationInsights`` and ``Seq``. All of which can be wrapped in an ``Async`` sink to avoid blocking Vonk when waiting for the sink to process the log statements.
 
-RollingFile
-^^^^^^^^^^^
-For the ``RollingFile`` sink, you can specify the location of the log files with the ``pathFormat`` argument.
-Please include the ``{Date}``, ``{Hour}`` or ``{HalfHour}`` placeholder, so Serilog can add date and time
-information to your filename.
+Console
+^^^^^^^
 
-*	Navigate to your Vonk working directory.
-*	Open :code:`logsettings.json` in a text editor and find the ``WriteTo`` setting::
+The Console sink will write to your shell.
+
+*	Find the ``WriteTo`` setting::
 
 		"WriteTo": [
-			{ "Name": "ColoredConsole" },
 			{
-				"Name": "RollingFile",
-				"Args": { "pathFormat": "c:/temp/vonk-{Date}.log" }
+				"Name": "Async",
+				"Args": {
+					"configure": [
+						{
+						"Name": "Console",
+						"Args": {
+							"restrictedToMinimumLevel": "Information",
+							"outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} {UserId} {Username} [{Level}] [ReqId: {RequestId}] {Message}{NewLine}{Exception}"
+						}
+						}
+					]
+				}
 			},
+			{
+				//Settings for other sinks
+			}
 
-*	Under ``RollingFile``, change the location of the logfiles by editing the value for ``pathFormat``.
-	For example::
-
-		{
-			"Name": "RollingFile",
-			"Args": { "pathFormat": "c:/logfiles/vonk-{Hour}.log" }
-		},
-
-You can also limit the size of the log file, which is 1GB by default, with the ``fileSizeLimitBytes`` option.
-If you set the value for this argument to ``null``, you remove the limit. Serilog retains a maximum of 31 files.
-This value can also be removed or changed, by providing the ``retainedFileCountLimit`` argument::
+The Console is notoriously slow at processing log statements, so it is recommended to limit the number of statements for this sink. Use the ``restrictedToMinimumLevel`` to do so.
```

Comment:
Someone mentioned to use the PowerShell window instead of a MS-DOS command window for faster processing.  Would it be useful to mention that here?

Reply:
Yes, but that still does not keep up if you put Vonk under stress :-)

Reply:
Still might be worth mentioning though.